app/models/project.rb

```diff
@@ -25,6 +27,12 @@ def trust
     @trust || retrieve_trust
   end
 
+  def caseworker_assigned_at=(value)
+    raise UpdateValueNotAllowedError, "Cannot update the value of `caseworker_assigned_at`" if caseworker_assigned_at.present?
```

Comment:
Interested to know what we think of this... I think a "cleaner" solution may have been:
```ruby
return if caseworker_assigned_at.present?
```
But I can imagine that being a nightmare for any future developers, wondering why the value isn;t updating as they expect 🤔